impeller/renderer/pipeline.h

```diff
@@ -66,6 +66,8 @@ class Pipeline {
  protected:
   Pipeline(std::weak_ptr<PipelineLibrary> library, T desc);
 
+  bool IsPipelineLive() const { return static_cast<bool>(library_.lock()); }
```

Comment:
`IsPipelineLive` was added as part of this PR - remove it now that it's no longer being called

Reply:
done